tests/manual/publishing-dialog.Rmd

```diff
@@ -33,3 +33,46 @@ Success:
 
 1. No errors in console
 1. `nrow(rsconnect::accounts("shinyapps.io"))` returns 1 or greater.
+
+# Apps
```

Comment:
The combinatorics can quickly explode..

1. having renv.lock
2. without renv.lock
3. environment that lacks some referenced package (error)

It's probably not worth enumerating every "it's an error" case, but the "missing package" error felt common enough to call out.

Reply:
FWIW those are in `dependencies.Rmd`